ark/utils/tiff_utils.py

```diff
@@ -46,6 +47,16 @@ def read_mibitiff(file, channels=None):
             # read image data
             img_data.append(page.asarray())
 
+    # make sure all passed channels were found
+    if channels is not None:
+        channel_names = [return_channel[1] for return_channel in return_channels]
+        found_channels = [channel not in channel_names for channel in channels]
```

Comment:
We can do verify_in_list for this right?

Reply:
totally.  I do think it should be an IndexError instead of a ValueError in this case, so I'll wrap it in a try/except block and re-raise the error as an IndexError.

Reply:
Sounds good. Other option would be to add option to specify error type as an argument to verify_in_list. I'm cool with either